src/binding/transaction_log_file_posix.cpp

```diff
@@ -128,6 +145,13 @@ int64_t TransactionLogFile::writeToFile(const void* buffer, uint32_t size, int64
 	return static_cast<int64_t>(::write(this->fd, buffer, size));
 }
 
+MemoryMap::MemoryMap(void* map, uint32_t mapSize) : map(map), mapSize(mapSize) {}
```

Comment:
This constructor should go in `transaction_log_file.cpp` or inlined in `transaction_log_file.h`.